### PR TITLE
Lazy load sealed secret certificates

### PR DESCRIPTION
**Pull request by teodor-pripoae**

### CODE REVIEW

**Comment by moshloop** [comment]
:neutral_face: commit 70853d6b85bea29d01668c3cdfc8edf84b93313e had some failures or skipped tests. **Is it OK?**
| Class| Message | Result |
|------|---------|--------|
| **test.thanos** | `Must specify a thanos server under e2e.server in client mode` | :white_circle: |
| **test.Elasticsearch** | `elastichsearch is not installed or enabled` | :white_circle: |
| **test.registry-creds** | `registry credentials not configured or disabled` | :white_circle: |

**Comment by moshloop** [comment]
:neutral_face: commit 70853d6b85bea29d01668c3cdfc8edf84b93313e had some failures or skipped tests. **Is it OK?**
| Class| Message | Result |
|------|---------|--------|
| **test.thanos** | `Must specify a thanos server under e2e.server in client mode` | :white_circle: |
| **test.registry-creds** | `registry credentials not configured or disabled` | :white_circle: |
| **test.Elasticsearch** | `elastichsearch is not installed or enabled` | :white_circle: |

File: pkg/platform/platform.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/moshloop/platform-cli/templates"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"gopkg.in/flanksource/yaml.v3"
+	yaml "gopkg.in/yaml.v2"
```
Comment (moshloop) [inline pkg/platform/platform.go:39]: is this intentional ?

Comment (teodor-pripoae) [reply pkg/platform/platform.go]: No, it got autochanged by IDEA.

**Comment by moshloop** [comment]
:neutral_face: commit 0c77c307ff6bdced997bb4c2178441cddde64759 had some failures or skipped tests. **Is it OK?**
| Class| Message | Result |
|------|---------|--------|
| **test.thanos** | `Must specify a thanos server under e2e.server in client mode` | :white_circle: |
| **test.Elasticsearch** | `elastichsearch is not installed or enabled` | :white_circle: |
| **test.registry-creds** | `registry credentials not configured or disabled` | :white_circle: |
| **test.local-path-storage** | `local-path-storage/create-pvc-e298f361-190b-4616-aec3-87d461d3a5ae=Pending  ` | :x: |

**Comment by moshloop** [comment]
:neutral_face: commit 0c77c307ff6bdced997bb4c2178441cddde64759 had some failures or skipped tests. **Is it OK?**
| Class| Message | Result |
|------|---------|--------|
| **test.registry-creds** | `registry credentials not configured or disabled` | :white_circle: |
| **test.Elasticsearch** | `elastichsearch is not installed or enabled` | :white_circle: |
| **test.thanos** | `Must specify a thanos server under e2e.server in client mode` | :white_circle: |

**Pull request merged**